frontend/src/pages/ClientVenues.jsx

```diff
@@ -1,8 +1,55 @@
+import React from "react";
+import Navbar from "../components/Navbar";
+import Saddledome from "../assets/Saddle.jpg"
+import TMobile from "../assets/T-Mobile2.png"
+import Amalie from "../assets/Amalie.jpg"
+
+const BUTTON_CONTAINER = "bg-primary w-64 flex flex-col items-center text-xl my-8 mx-auto py-2  rounded-lg hover:text-dark ease-in duration-300 font-bold"
+
+
 function ClientVenues() {
 	return ( 
-		<div className="bg-dark min-h-screen pt-36 text-white">
-			<h1>client venues</h1>
-		</div>
+		<>
+			<Navbar />
+			<div className = "flex flex-row space-x-50 justify-center items-center ">
```

Comment:
for a lot of the styling you wrote ``className = "... styling"`` but we should be doing
``className="...styling"`` to maintain consistency

Reply:
I think you also forgot to include the ``AVAILABLE VENUES`` text on the page which should be in an h1